docs/ref/contrib/gis/install/spatialite.txt

```diff
@@ -7,9 +7,7 @@ spatial database.
 
 First, check if you can install SpatiaLite from system packages or binaries.
 
-For example, on Debian-based distributions, try to install the
-``spatialite-bin`` package. For distributions that package SpatiaLite 4.2+,
-install ``libsqlite3-mod-spatialite``.
+For example: With Debian-based distributions that package SpatiaLite 4.2+ in their recent releases (e.g Debian Stretch, Ubuntu 18.04, etc.), install ``libsqlite3-mod-spatialite``. For older releases, try to install the ``spatialite-bin`` package.
```

Comment:
IMO previous wording and sentence construction was correct, so maybe:
```
For example, on Debian-based distributions that package SpatiaLite 4.2+, try to
install the ``libsqlite3-mod-spatialite`` package. For older releases install
``spatialite-bin``.
```
Please wrap docs at 79 chars.